app/routes/_gcn.circulars/circulars.lib.ts

```diff
@@ -36,6 +36,20 @@ type SubjectMatcher = [RegExp, (match: RegExpMatchArray) => string]
 
 const subjectMatchers: SubjectMatcher[] = [
   [/GRB[.\s_-]*(\d{6}[a-z|.]\d*)/i, ([, id]) => `GRB ${id.toUpperCase()}`],
+  [/GRB[.\s_-]*(\d{6}[a-z|.]\d*):/i, ([, id]) => `GRB ${id.toUpperCase()}`],
+  [
+    /IBAS Alert No[.](\d{4})/i,
+    ([, id]) => `IBAS Alert No. ${id.toUpperCase()}`,
+  ],
+  [/IBAS Alert n[.](\d{4})/i, ([, id]) => `IBAS Alert No. ${id.toUpperCase()}`],
+  [
+    /IBAS Alert n[.] (\d{4})/i,
+    ([, id]) => `IBAS Alert No. ${id.toUpperCase()}`,
+  ],
+  [/XRF(\d{6})/i, ([, id]) => `XRF ${id.toUpperCase()}`],
+  [/XRF (\d{6})/i, ([, id]) => `XRF ${id.toUpperCase()}`],
```

Comment:
```suggestion
  [/XRF\s*(\d{6})/i, ([, id]) => `XRF ${id.toUpperCase()}`],
```
Also: `id.toUpperCase()` will have no effect here, because the group `(\d{6})` matches only digits and not letters.